e2e/cypress/integration/api/iam/01_projects_api.spec.ts

```diff
@@ -1,157 +1,164 @@
-const avengersProject = {
-  id: `avengers-project-${Cypress.moment().format('MMDDYYhhmm')}`,
-  name: 'Test Avengers Project'
-};
-
-const xmenProject = {
-  id: `xmen-project-${Cypress.moment().format('MMDDYYhhmm')}`,
-  name: 'Test X-men Project'
-};
-
-const avengersRule = {
-  id: 'avengers-rule-1',
-  name: 'first rule of avengers project',
-  type: 'NODE',
-  project_id: avengersProject.id,
-  conditions: [
-    {
-      attribute: 'CHEF_ORGANIZATION',
-      operator: 'EQUALS',
-      values: ['avengers']
-    }
-  ]
-};
-
-const xmenRule = {
-  id: 'xmen-rule-1',
-  name: 'first rule of xmen project',
-  type: 'NODE',
-  project_id: xmenProject.id,
-  conditions: [
-    {
-      attribute: 'CHEF_ORGANIZATION',
-      operator: 'EQUALS',
-      values: ['xmen']
-    }
-  ]
-};
-
 if (Cypress.env('IAM_VERSION') === undefined) {
   // assume local env is on IAM v2.0 since that's the default setting
   Cypress.env('IAM_VERSION', 'v2.0');
 }
 const describeIfIAMV2p1 = Cypress.env('IAM_VERSION') === 'v2.1' ? describe : describe.skip;
 
 describeIfIAMV2p1('projects API: applying project', () => {
+  const cypressPrefix = 'cypress-test';
+  const avengersProject = {
+    id: `${cypressPrefix}-avengers-project-${Cypress.moment().format('MMDDYYhhmm')}`,
+    name: 'Test Avengers Project'
+  };
+
+  const xmenProject = {
+    id: `${cypressPrefix}-xmen-project-${Cypress.moment().format('MMDDYYhhmm')}`,
+    name: 'Test X-men Project'
+  };
+
+  const avengersRule = {
+    id: 'avengers-rule-1',
+    name: 'first rule of avengers project',
+    type: 'NODE',
+    project_id: avengersProject.id,
+    conditions: [
+      {
+        attribute: 'CHEF_ORGANIZATION',
+        operator: 'EQUALS',
+        values: ['avengers']
+      }
+    ]
+  };
+
+  const xmenRule = {
+    id: 'xmen-rule-1',
+    name: 'first rule of xmen project',
+    type: 'NODE',
+    project_id: xmenProject.id,
+    conditions: [
+      {
+        attribute: 'CHEF_ORGANIZATION',
+        operator: 'EQUALS',
+        values: ['xmen']
+      }
+    ]
+  };
+  let adminIdToken = '';
+
   before(() => {
     cy.adminLogin('/').then(() => {
-      const admin = JSON.parse(<string>localStorage.getItem('chef-automate-user'));
+      adminIdToken = JSON.parse(<string>localStorage.getItem('chef-automate-user')).id_token;
 
-    // Cypress recommends state cleanup in the before block to ensure
-    // it gets run every time:
-    // tslint:disable-next-line:max-line-length
-    // https://docs.cypress.io/guides/references/best-practices.html#Using-after-or-afterEach-hooks
-    cleanupTestProjects(admin.id_token);
+      // Cypress recommends state cleanup in the before block to ensure
+      // it gets run every time:
+      // tslint:disable-next-line:max-line-length
+      // https://docs.cypress.io/guides/references/best-practices.html#Using-after-or-afterEach-hooks
+      cleanupTestProjects(adminIdToken);
+      cy.cleanupProjectsByIDPrefix(adminIdToken, cypressPrefix);
```

Comment:
Actual change 2: cleanup all the projects.

Reply:
Will we ever need different ID prefixes? We could make `cypress-test` available a global config value in cypress.json and simplify some of these cleanup functions